adoc-mode.el

```diff
@@ -1556,6 +1556,19 @@ Subgroups of returned regexp:
 
 
 ;;;; font lock keywords
+(defsubst adoc-kwf-search (regexp &optional bound noerror count)
```

Comment:
Why `kwf` and not just `kw`?

Reply:
Because `adoc-kwf-search` does not return a font-lock keyword but it is an auxiliary Function much like `adoc-kwf-std`.

Reply:
I get this, but I'm struggling to understand what does `kwf` stands for. :-)

Reply:
I assume that `-kwf-` stands for keyword function.
Those are functions that are used in the keyword generators `-kw-`.

Reply:
Well, here it's not even a function, as I see it's being inlined. :-) Anyways, I don't feel strongly about this naming - I just find it somewhat confusing.

Reply:
Reasoning for the naming:
1. Keep changes small
2. Keep consistency